modules/test/playwright/tests/users-admin-web/main/organizations.spec.ts

```diff
@@ -0,0 +1,236 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../../fixtures/apiHelpersTest';
+import {dataApiHelpersTest} from '../../../fixtures/dataApiHelpersTest';
+import {loginTest} from '../../../fixtures/loginTest';
+import {usersAndOrganizationsPagesTest} from '../../../fixtures/usersAndOrganizationsPagesTest';
+import {createCategories} from '../../../helpers/CreateCategories';
+import {getRandomInt} from '../../../utils/getRandomInt';
+import {waitForAlert} from '../../../utils/waitForAlert';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+	loginTest(),
+	usersAndOrganizationsPagesTest
+);
+
+test(
+	'Add multiple suborganizations to parent organization',
+	{tag: '@LPD-57824'},
+	async ({apiHelpers, usersAndOrganizationsPage}) => {
+		const parentOrganization =
+			await apiHelpers.headlessAdminUser.postOrganization({
+				name: 'Parent Organization' + getRandomInt(),
+			});
+
+		const subOrganizations = [];
+
+		for (let count = 1; count <= 5; count++) {
+			const subOrganization =
+				await apiHelpers.headlessAdminUser.postOrganization({
+					name: `Suborganization Name ${count}`,
+					parentOrganization: {
+						externalReferenceCode:
+							parentOrganization.externalReferenceCode,
+					},
+				});
+
+			subOrganizations.push(subOrganization);
+		}
+
+		await usersAndOrganizationsPage.goToOrganizations();
+
+		await usersAndOrganizationsPage.organizationsTable
+			.valueLink(parentOrganization.name)
+			.click();
+
+		for (const subOrganization of subOrganizations) {
+			await expect(
+				usersAndOrganizationsPage.organizationsTable.cell(
+					subOrganization.name
+				)
+			).toBeVisible();
+		}
+
+		await usersAndOrganizationsPage.goToOrganizations();
+
+		await usersAndOrganizationsPage.organizationsTable.search(
+			'Suborganization'
+		);
+
+		for (const subOrganization of subOrganizations) {
+			await expect(
+				usersAndOrganizationsPage.organizationsTable.cell(
+					subOrganization.name
+				)
+			).toBeVisible();
+		}
+	}
+);
+
+test(
+	'Add, view and delete organization, add comment, and assign global category to it.',
+	{tag: '@LPD-57824'},
+	async ({
+		apiHelpers,
+		editOrganizationPage,
+		page,
+		usersAndOrganizationsPage,
+	}) => {
+		await page.on('dialog', (dialog) => dialog.accept());
+
+		const orgName = 'Organization' + getRandomInt();
+
+		const deleteOrganization = async () => {
```

Comment:
for the future: probably is better to have the orgName passed as parameter to this function in this case